calour/amplicon_experiment.py

```diff
@@ -254,3 +254,13 @@ def split_taxonomy(self, field='taxonomy', sep=';',
             the column names for the new columns split from ``field``
         '''
         self.feature_metadata[names] = self.feature_metadata[field].str.split(sep, expand=True)
+        # return so you can chain the functions
+        return self
+
+    def find_lowest_taxonomy(self, field='taxonomy', new_field='taxa'):
```

Comment:
maybe worth adding a docstring (what the function does, parameters, etc.)
or is it internal (and then maybe add _ before name)?
Do we use this new function somewhere in calour? or is it just for analysis